calculation/sha_calc/sha_calc/directivity/bea20/benchmark_tests.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+from pathlib import Path
```

Comment:
same here, I know it's just a part of the benchmark test but might as well keep the consistency :+1: